frontend/src/scenes/activity/explore/EventDetails.tsx

```diff
@@ -145,6 +147,14 @@ export function EventDetails({ event, tableProps }: EventDetailsProps): JSX.Elem
             label: 'Conversation',
             content: (
                 <div className="mx-3 -mt-2 mb-2 space-y-2">
+                    {event.properties.$session_id ? (
+                        <div className="flex flex-row items-center gap-2">
+                            <Link to={urls.replay(undefined, undefined, event.properties.$session_id)}>
+                                <IconOpenInNew />
+                                View session recording
+                            </Link>
+                        </div>
+                    ) : null}
```

Comment:
What Paul is suggesting would basically look like this:
```suggestion
                    {mightHaveRecording(event.properties) && (
                        <ViewRecordingButton sessionId={event.properties.$session_id} />
                    )}
```